keel-api/src/test/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigControllerTests.kt

```diff
@@ -141,7 +141,12 @@ internal class DeliveryConfigControllerTests : JUnit5Minutests {
         |      application: someapp
         |- name: prod
         |  constraints:
-        |  - environment: test
+        |  - type: depends-on
+        |    environment: test
+        |  - type: time-window
+        |    windows:
+        |      - hours: 6-18
+        |      - days: mon-fri
```

Comment:
Should this yaml be:
```
windows:
- hours: 6-18
  days: mon-fri
```
(no dash by days) to mean that those go together as one window?

Reply:
Ooh yes, otherwise it will be 2 separate windows

Reply:
good catch, I'm so bad at yaml!